example/src/components/RoomParticipant.tsx

```diff
@@ -0,0 +1,94 @@
+import { BrandColors, AdditionalColors, TextColors } from '@colors';
+import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { getShortUsername } from '@utils';
+import React from 'react';
+import { View, StyleSheet, ViewStyle } from 'react-native';
+
+import { Icon } from './Icon';
+import { NoCameraView } from './NoCameraView';
+import { Typo } from './Typo';
+
+type RoomParticipantProps = {
+  participant: Membrane.Participant;
+  pStyle: ViewStyle[];
+};
+
+export const RoomParticipant = ({
+  participant,
+  pStyle,
+}: RoomParticipantProps) => {
+  return (
+    <View style={pStyle}>
+      {!participant.tracks.filter((t) => t.type === 'Video').length ||
+      !participant.tracks.find((t) => t.type === 'Video')!.metadata.active ? (
```

Comment:
I'd extract this check into a small function in this component, it would be more readable. Like `participantHasVideo()`. Or even a boolean variable. You can also get video track earlier, make an if and avoid `!` operator. Using `!` operator is a bad practice. Also you'd avoid repeating `participant.tracks.find((t) => t.type === 'Video')`

Reply:
Done.